src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -57,4 +57,5 @@
   <StringProperty Name="DebugSymbols" DisplayName="Debug symbols" Visible="False"/>
   <EnumProperty Name="FileAlignment" DisplayName="File Alignment" Visible="False"/>
   <StringProperty Name="BaseAddress" DisplayName="Base address" Visible="False"/>
+  <StringProperty Name="FullPath" DisplayName="Project Folder" ReadOnly="True"/>
```

Comment:
Where does this property come from? There is no MSBuild property called "FullPath" that we populate - is this specially handled?